using_images/s2i_images/wildfly.adoc

```diff
@@ -0,0 +1,146 @@
+[[using-images-s2i-images-wildfly]]
+= Wildfly
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+
+toc::[]
+
+== Overview
+{product-title} provides
+xref:../../architecture/core_concepts/builds_and_image_streams.adoc#source-build[S2I]
+enabled Wildfly images for building and running Java applications.
+ifdef::openshift-origin[]
+The https://github.com/openshift-s2i/s2i-wildfly[Wildfly S2I builder image]
+endif::openshift-origin[]
+ifdef::openshift-enterprise[]
+The Wildfly S2I builder image
+endif::openshift-enterprise[]
+assembles your application source with any required dependencies to create a
+new image containing your Java application. This resulting image can be run
+either by {product-title} or by Docker.
+
+== Versions
+Currently, {product-title} supports version
+https://github.com/openshift-s2i/s2i-wildfly/tree/master/10.1[10.1], and
+https://github.com/openshift-s2i/s2i-wildfly/tree/master/10.0[10.0] of Wildfly.
+
+== Images
+
+These images are available on DockerHub. To download them:
+
+----
+$ docker pull openshift/wildfly-101-centos7
+$ docker pull openshift/wildfly-100-centos7
+----
+
+To use these images, you can either access them directly from these
+xref:../../architecture/infrastructure_components/image_registry.adoc#architecture-infrastructure-components-image-registry[image
+registries], or push them into your
```

Comment:
will reword, at least in this case it's simpler since they only exist on dockerhub.